### ISSUE TITLE
Enhancement: can you include a non-static version of learner.view_top_losses?

### ISSUE THREAD

**Issue by ProbonoBonobo** [amaiya/ktrain#253]
From the medium tutorial page:

> By invoking `learner.view_top_losses(preproc, n=3)` after training, we can view the top n examples in the validation set that are the most severely misclassified. This can shed light on how to improve your model or data-processing pipeline and whether to prune the dataset of “garbage” data. 

This function, as promised, is _extremely_ helpful for eyeballing outliers in the training data, I'm not sure how I functioned without it. One minor point of feedback, though: for my purposes, it would be even more helpful it were possible to invoke a related method (`get_top_losses` maybe?) that returns a list of integers corresponding to the indices of these 'most severely misclassified' training examples. This would enable users to iterate over those items -- perhaps in combination with the (also insanely helpful) `predictor.explain` method.

Edit: oops I don't think I meant to use the phrase "non-static." I was thinking of a situation that in Java is called void return types, but I'm not sure what this would be called more generally.

**Comment by amaiya** [comment]
Hi,

The `view_top_losses` method simply calls `top_losses`, which returns a list of tuples containing the information displayed by `view_top_losses`.  So,  you can call `learner.top_losses` to get this information. The first entry of each tuple is the ID.

In a future version of ktrain, this method may be changed to return a list of dictionaries instead of list of tuples to better facilitate such  use cases.

**Comment by ProbonoBonobo** [comment]
Thank you! I hadn't seen this method referred to in the API. Thanks also for the heads up that the interface might change in the future.
